shuup/admin/modules/orders/static_src/create/view/lines.js

```diff
@@ -110,6 +111,15 @@ export function renderOrderLines(store, shop, lines) {
                 renderNumberCell(store, line, line.total, "total", canEditPrice)
             ])
         ];
+        if (line.type === "product" && line.product ){
+            editCell = m("div.line-cell.edit",
+                m("button.btn.btn-sm.text-info", {
```

Comment:
Note: This is done as a button instead of a link to prevent alignment issues with delete button